third_party/BUILD

```diff
@@ -46,8 +44,6 @@ filegroup(
 filegroup(
```

Comment:
@philwo @janakdr this change probably isn't OK? We still want to include the sources.

Reply:
@cushon can you elaborate? Is that because of LGPL license?
The sources must be in tree? Can we merge from source-JAR downloaded from Central?

Reply:
I don't see anything now. Are these changes reverted?

Reply:
Yes, I uploaded a new version of the PR. I restored the GPL source packaging. But now the jars are provided and included in `embedded_tools`:

```
  $ bazel build src:embedded_tools
INFO: Analysed target //src:embedded_tools (1 packages loaded).
INFO: Found 1 target...
Target //src:embedded_tools up-to-date:
  bazel-genfiles/src/embedded_tools.zip
INFO: Elapsed time: 0.279s, Critical Path: 0.00s

  $ unzip -t bazel-genfiles/src/embedded_tools.zip | grep checker_framework
    testing: third_party/checker_framework_dataflow/dataflow-2.2.2-sources.jar   OK
    testing: third_party/checker_framework_javacutil/javacutil-2.2.2-sources.jar   OK

```